app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -430,6 +430,7 @@ type CategoryProducts @doc(description: "The category products object returned i
 
 input ProductFilterInput @doc(description: "ProductFilterInput defines the filters to be used in the search. A filter contains at least one attribute, a comparison operator, and the value that is being searched for.") {
     name: FilterTypeInput @doc(description: "The product name. Customers use this name to identify the product.")
+    entity_id: FilterTypeInput @doc(description: "The product entity id. Customers use this id to identify the product.")
```

Comment:
Hi @paliarush,
Please approve adding filtering by `entity_id` because in new code we considered that `entity_id` is surrogate key and we need to use just `sku`

@ronak2ram 
After approving need to implement support of Staging functionality
Because we need to find product by `row_id` (instead of finding by `entity_id`) if Staging is enabled and we have some revision for Product

Reply:
No

Reply:
@naydav PWA needs this feature and we should added it

Reply:
Maybe we need to communicate with PWA team and describe that is wrong way